File: solution_zzj/6181._Length_of_the_Longest_Alphabetical_Continuous_Substring.cc
class Solution {
public:
    int longestContinuousSubstring(string s) {
        map<char, int> m;
        string alp = "abcdefghijklmnopqrstuvwxyz";
        int i = 0, ans = 0, cur = 1;
        for (auto c: alp)
            m[c] = i++;
        for (int i = 0; i < s.size() - 1; i++) {
            if (m[s[i+1]] - m[s[i]] == 1) {
                cur++;
            } else {
                ans = max(ans, cur);
                cur = 1;
            }
        }
        return max(ans,cur);
    }
};

File: Weijia/6181. Length of the Longest Alphabetical Continuous Substring.py
class Solution(object):
    def longestContinuousSubstring(self, s):
        if len(s)<2:
            return len(s)
        else:
            i,j=0,1
            n=1
            l=[1]
            while i <len(s) and j< len(s):
                if s[i]+s[j] in "abcdefghijklmnopqrstuvwxyz":
                    n+=1
                else:
                    l.append(n)
                    n=1
                i+=1
                j+=1
                l.append(n)
            return max(l)
            
                    
